{"version":1,"title":"linux-next test error: lost connection to test machine (2)","id":"83d3947a32dfc8185fe2a208345612f473a6e68a","discussions":["http://b/441638788"],"crashes":[{"title":"linux-next test error: lost connection to test machine","kernel-config":"/text?tag=KernelConfig\u0026x=ad4ad12942244ded","kernel-source-git":"https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/log/?id=8cd53fb40a304576fa86ba985f3045d5c55b0ae3","kernel-source-commit":"8cd53fb40a304576fa86ba985f3045d5c55b0ae3","syzkaller-git":"https://github.com/google/syzkaller/commits/443c11c765c0bf86d91595bf5edd50af4476fdc6","syzkaller-commit":"443c11c765c0bf86d91595bf5edd50af4476fdc6"},{"title":"linux-next test error: lost connection to test machine","kernel-config":"/text?tag=KernelConfig\u0026x=64e3327f3e06f2bc","kernel-source-git":"https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/log/?id=8cd53fb40a304576fa86ba985f3045d5c55b0ae3","kernel-source-commit":"8cd53fb40a304576fa86ba985f3045d5c55b0ae3","syzkaller-git":"https://github.com/google/syzkaller/commits/e12e5ba469cd05aa76330394486b1f2a24dd4ef6","syzkaller-commit":"e12e5ba469cd05aa76330394486b1f2a24dd4ef6"}]}